edelphi/src/main/java/fi/metatavu/edelphi/pages/panel/admin/report/util/ChartModelProvider.java

```diff
@@ -617,7 +638,7 @@ public static Chart createBubbleChart(String chartCaption, String xLabel, List<S
 
     // X-Axis
 
-    Map<Double, String> xTickLabelMap = new HashMap<Double, String>();
+    Map<Double, String> xTickLabelMap = new HashMap<>();
     for (int i = 0, l = xTickLabels.size(); i < l; i++) {
       xTickLabelMap.put(new Double(i), xTickLabels.get(i));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this "Double" constructor [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2129)